app/controllers/cloud_volume_snapshot_controller.rb

```diff
@@ -67,6 +71,42 @@ def show_list
     process_show_list
   end
```

Comment:
this problem is endemic to the entire front-end. it would be nice to have a universal approach.